hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java

```diff
@@ -460,9 +460,23 @@ public void provideMetrics(MetricsRegistry registry) {
     public void start() {
         logger.finest("Starting OperationService");
 
-        ManagedExecutorService asyncExecutor = nodeEngine.getExecutionService().register(
-                ExecutionService.ASYNC_EXECUTOR, Runtime.getRuntime().availableProcessors(),
-                ASYNC_QUEUE_CAPACITY, ExecutorType.CONCRETE);
+        setInvocationContext();
+
+        invocationMonitor.start();
+        operationExecutor.start();
+        asyncResponseHandler.start();
+        slowOperationDetector.start();
+    }
+
+    public void setInvocationContext() {
```

Comment:
Sure, I will rename it. It is public because HotRestartService accesses it during force start.